onnxruntime/python/tools/kernel_explorer/kernel_explorer_interface.h

```diff
@@ -78,3 +82,22 @@ class IKernelExplorer {
   StreamT stream_{0};
   int repeats_{100};
 };
+
+pybind11::module GetKernelExplorerModule();
+
+class KernelExplorerInit {
+ public:
+  explicit KernelExplorerInit(void (*init_func)(pybind11::module module)) {
+    init_func(GetKernelExplorerModule());
+  }
+};
+
+#define KE_REGISTER_IMPL(unique_id, module_name)                                    \
+  static void KeInitFunc##unique_id(pybind11::module module_name);                  \
+  static const KernelExplorerInit kKeInitializer##unique_id{KeInitFunc##unique_id}; \
```

Comment:
`kKeInitializer`, do you want double `k` here?

Reply:
yeah, because it is a `const`